src/content/docs/business-observability/troubleshoot-pathpoint.mdx

```diff
@@ -0,0 +1,200 @@
+
+<img
+    title="Active Canary Filter"
+    alt="Active Canary Filter"
+    src={activeCanaryFilter}
+/>
+
+Notice that enabling a step also enables the corresponding touchpoints tied to the step.
+
+## Flame filter
+
+The Flame Filter highlights the most problematic “steps and touchpoints” with a configurable time window and percentile threshold. 
+When using this filter, you see the worst-performing touchpoints highlighted in red, making it easy to identify the areas that require attention. For example, the filter can help you identify touchpoints that are not meeting the threshold percentage set in the tuning.
+This is very useful in situations where things look okay now, but might have had a pattern of errors or latency in the recent past. 
+
+Before you use this special Pathpoint filter, you must enable **Flame Filter Background Script**.
+
+### Enable Flame Filter Background Script
+
+To enable the Flame Filter Background Script:
+
+<Steps>
+
+<Step>
+
+Navigate to the Pathpoint screen and click on the three lines in the top left part of the window.
+
+<img
+  title="Navigate to Pathpoint config"
+  alt="Navigate to Pathpoint config"
+  src={pathpointConfig}
+/>
+
+</Step>
+
+<Step>
+
+Here, select **Credentials and General Configuration**.
+
+<img
+  title="Pathpoint Credentials and General Configuration"
+  alt="Pathpoint Credentials and General Configuration"
+  src={generalConfig}
+/>
+
+</Step>
+
+<Step>
+
+On the **Credentials and General Configuration** dialog box window, you see a few options asking you to fill **Ingest License** key, as well as a **User API Key**, and options to ** Enable Pathpoint Logging** and **Enable Flame Filter Background Script**.
+
+Once you've filled in the required data and checked both options, click **Save/Update**.
+
+<img
+  title="Save/Update Pathpoint Credentials"
+  alt="Save/Update Pathpoint Credentials"
+  src={credentials}
+/>
+
+This takes you back to main Pathpoint window.
+
+</Step>
+
+<Step>
+
+From the main Pathpoint screen, again click on the three lines in the top left part of the window.
+
+<img
+  title="Navigate to Pathpoint config"
+  alt="Navigate to Pathpoint config"
+  src={pathpointConfig}
+/>
+
+</Step>
+
+<Step>
+
+Now, click **Credentials and General Configuration**.
+
+<img
+  title="Pathpoint Credentials and General Configuration"
+  alt="Pathpoint Credentials and General Configuration"
+  src={generalConfig}
+/>
+
+</Step>
+
+<Step>
+
+Finally, click **Install/Update Job**.
+
+<img
+  title="Install/Update Job"
+  alt="Install/Update Job"
+  src={installJob}
+/>
+
+</Step>
+
+Now, both **Pathpoint Logging**, and **Flame Filter Background Script** are enabled.
+
+</Steps>
+
+Once you've enabled the **Flame Filter Background Script**, you can activate the Flame filter by just clicking on the Flame Symbol.
```

Comment:
I'd add the logo instead of describing it. That way, we can also remove the screenshot below.